C++ compatibility

The C++ compatibility is broken due to some data structures for complex numbers.

```console
/usr/local/include/LAGraph.h:572:23: error: expected initializer before ‘LAGraph_ComplexFP64_1’
 extern double complex LAGraph_ComplexFP64_1;
                       ^~~~~~~~~~~~~~~~~~~~~
/usr/local/include/LAGraph.h:573:23: error: expected initializer before ‘LAGraph_ComplexFP64_0’
 extern double complex LAGraph_ComplexFP64_0;
                       ^~~~~~~~~~~~~~~~~~~~~
```

Interestingly, my IDE shows `Use of unknown builtin '__builtin_complex'` errors for `CMPLX`, `BOOL`, `ONE`, and `T` in `LAGraph_complex.h` but builds it regardless (as it's perfectly valid C code).

For the long term, we should decide how to handle C++ compatibility. If it's important, we should set up some tests to check it, otherwise, we should clarify the lack of support in the README.

--- a/Include/LAGraph.h
+++ b/Include/LAGraph.h
@@ -569,8 +569,8 @@ extern GrB_Monoid
     
 extern GrB_Semiring LAGraph_PLUS_TIMES_ComplexFP64 ;
 
-extern double complex LAGraph_ComplexFP64_1  ;
-extern double complex LAGraph_ComplexFP64_0 ;
+extern double _Complex LAGraph_ComplexFP64_1 ;
+extern double _Complex LAGraph_ComplexFP64_0 ;
 
 GrB_Info LAGraph_Complex_init ( ) ;
 GrB_Info LAGraph_Complex_finalize ( ) ;

